src/prospectScraper/Program.cs

```diff
@@ -39,720 +33,27 @@ static void Main(string[] args)
                 {
                     case "bb":
                         Console.WriteLine("Running Big Board");
-                        RunTheBigBoards(parseDate);
+                        prospectScraper.RunTheBigBoards(parseDate);
+                        prospectScraper.RunTheBigBoards(parseDate);
                         break;
                     case "md":
                         Console.WriteLine("Running Mock Draft");
-                        RunTheMockDraft(parseDate);
+                        prospectScraper.RunTheMockDraft(parseDate);
+                        prospectScraper.RunTheMockDraft(parseDate);
                         break;
                     case "all":
                         Console.WriteLine("Running Big Board and Mock Draft");
-                        RunTheBigBoards(parseDate);
-                        RunTheMockDraft(parseDate);
+                        prospectScraper.RunTheBigBoards(parseDate);
+                        prospectScraper.RunTheMockDraft(parseDate);
                         break;
                     default:
-                        Console.WriteLine("Input argument of " + context + " not recognized.  Please try running again.");
-                        RunTheBigBoards(parseDate);
+                        Console.WriteLine("Input argument of " + s + " not recognized. Please try running again.");
```

Comment:
this line needs to be removed.  It references the old generic variable "s".

Reply:
Woah, something did not push right in this merge resolution. Taking a look now...

Reply:
but this is done